ansible/inventories/devnet-10/group_vars/all/all.yaml

```diff
@@ -0,0 +1,211 @@
+  el/genesis-config.yaml: |
+    chain_id: ${CHAIN_ID}
+    deposit_contract_address: "${DEPOSIT_CONTRACT_ADDRESS}"
+    mnemonic: ${EL_AND_CL_MNEMONIC}
+    el_premine:
+      "m/44'/60'/0'/0/0": 10000000000ETH
+      "m/44'/60'/0'/0/1": 10000000000ETH
+      "m/44'/60'/0'/0/2": 10000000000ETH
+      "m/44'/60'/0'/0/3": 10000000000ETH
+      "m/44'/60'/0'/0/4": 10000000000ETH
+      "m/44'/60'/0'/0/5": 10000000000ETH
+      "m/44'/60'/0'/0/6": 10000000000ETH
+      "m/44'/60'/0'/0/7": 10000000000ETH
+      "m/44'/60'/0'/0/8": 10000000000ETH
+      "m/44'/60'/0'/0/9": 10000000000ETH
+      "m/44'/60'/0'/0/10": 10000000000ETH
+      "m/44'/60'/0'/0/11": 10000000000ETH
+      "m/44'/60'/0'/0/12": 10000000000ETH
+      "m/44'/60'/0'/0/13": 10000000000ETH
+      "m/44'/60'/0'/0/14": 10000000000ETH
+      "m/44'/60'/0'/0/15": 10000000000ETH
+      "m/44'/60'/0'/0/16": 10000000000ETH
+      "m/44'/60'/0'/0/17": 10000000000ETH
+      "m/44'/60'/0'/0/18": 10000000000ETH
+      "m/44'/60'/0'/0/19": 10000000000ETH
+      "m/44'/60'/0'/0/20": 10000000000ETH
+      "m/44'/60'/0'/0/21": 10000000000ETH
+      "m/44'/60'/0'/0/22": 10000000000ETH
+      "m/44'/60'/0'/0/23": 10000000000ETH
+      "m/44'/60'/0'/0/24": 10000000000ETH
+      "m/44'/60'/0'/0/25": 10000000000ETH
+      "m/44'/60'/0'/0/26": 10000000000ETH
+      "m/44'/60'/0'/0/27": 10000000000ETH
+      "m/44'/60'/0'/0/28": 10000000000ETH
+      "m/44'/60'/0'/0/29": 10000000000ETH
+    el_premine_addrs: {}
+    genesis_timestamp: ${GENESIS_TIMESTAMP}
+    genesis_delay: ${GENESIS_DELAY}
+    slot_duration_in_seconds: ${SLOT_DURATION_IN_SECONDS}
+    deneb_fork_epoch: ${DENEB_FORK_EPOCH}
+
+  values.env: |-
+    export CHAIN_ID="{{ ethereum_genesis_chain_id }}"
+    export DEPOSIT_CONTRACT_ADDRESS="0x6f22fFbC56eFF051aECF839396DD1eD9aD6BBA9D"
+    export EL_AND_CL_MNEMONIC="{{ ethereum_genesis_mnemonic }}"
+    export CL_EXEC_BLOCK="0"
+    export DEPOSIT_CONTRACT_BLOCK="0x0000000000000000000000000000000000000000000000000000000000000000"
+    export NUMBER_OF_VALIDATORS=330000
+    export GENESIS_FORK_VERSION="0x10{{ ethereum_genesis_fork_version_suffix }}"
+    export ALTAIR_FORK_VERSION="0x20{{ ethereum_genesis_fork_version_suffix }}"
+    export BELLATRIX_FORK_VERSION="0x30{{ ethereum_genesis_fork_version_suffix }}"
+    export CAPELLA_FORK_VERSION="0x40{{ ethereum_genesis_fork_version_suffix }}"
+    export DENEB_FORK_VERSION="0x50{{ ethereum_genesis_fork_version_suffix }}"
+    export DENEB_FORK_EPOCH="256"
+    export WITHDRAWAL_TYPE="0x00"
+    export WITHDRAWAL_ADDRESS={{ ethereum_genesis_validator_bls_change_execution_address }}
+    export BEACON_STATIC_ENR=""
+    export GENESIS_TIMESTAMP={{ ethereum_genesis_timestamp }}
+    export GENESIS_DELAY={{ ethereum_genesis_timedelay }}
+    export SLOT_DURATION_IN_SECONDS=12
+    export MAX_CHURN=4
+    export EJECTION_BALANCE=31000000000
```

Comment:
didnt you want to set it down to 16 again?

Reply:
do we ?

Reply:
fine with or without, just thought of pointing it out